src/features/MentorPage/MentorList/MentorCard/Expanded/BasicInfo.tsx

```diff
@@ -14,7 +14,7 @@ export const BasicInfo = ({
   mentor: { name, age, region, isVacationing, statusMessage, languages },
 }: Props) => {
   const isMobile = useMobileMode();
-  const { t } = useTranslation();
+  const { t } = useTranslation('mentors');
```

Comment:
i like this namespace thing

Reply:
It is nice. You won't have to scroll through a massive translation file. And if we'd have a huge amount of translation keys it would also affect performance.